README.md

```diff
@@ -6,13 +6,18 @@ The ModelService Helm Chart proposal is accepted on June 10, 2025. Read more abo
 
 TL;DR:
 
-Active scearios supported
-- P/D disaggregation using deployments
-- P/D disaggregation using LeaderWorkerSets
+Active scearios supported:
+- P/D disaggregation
+- Multi-node inference, utilizing data parallelism
 - One pod per DP rank (in progress)
+- One pod per node (in progress)
```

Comment:
I suggest take out the "(in progress)". As far as we know modelservice supports both of these scenarios.

Reply:
Fixed.